packages/orchestrator/lib/clients/client.ts

```diff
@@ -254,17 +254,28 @@ export class OrchestratorClient {
         return this.execute(schedulingProps);
     }
 
-    public async executeOnEvent(props: ExecuteOnEventProps): Promise<ExecuteReturn> {
+    public async executeOnEvent(props: ExecuteOnEventProps): Promise<VoidReturn> {
         const { args, ...rest } = props;
         const schedulingProps = {
+            retry: { count: 0, max: 0 },
+            timeoutSettingsInSecs: {
+                createdToStarted: 30,
+                startedToCompleted: 5 * 60,
```

Comment:
like all our task timeout, this value is arbitrary. 5 mins is very generous imho